test_runlike.py

```diff
@@ -217,3 +217,11 @@ def setUpClass(cls):
 
     def test_no_name(self):
         self.dont_expect_substr("--name")
+
+class TestRunlikeNoName(BaseTest):
```

Comment:
You probably meant to call it `...NoLabels`, right?

Reply:
Yes… copy+paste error, fixed via amended commit (https://github.com/lavie/runlike/commit/da0e9cccf3a42c86d88d73bbee73f00429d616f0).